### PR TITLE
refactor(editor): Showing errors in input pane

### PR DESCRIPTION
**Pull request by cstuncsik**
Currently, output errors are also shown in the input pane which can be confusing 
image

The new approach is to tell the user a previous node has some errors and provide a link back to that node to see the output error
image

### CODE REVIEW

File: packages/editor-ui/src/plugins/i18n/locales/en.json [hunk 1]
```diff
@@ -470,6 +470,8 @@
 	"nodeErrorView.stack": "Stack",
 	"nodeErrorView.theErrorCauseIsTooLargeToBeDisplayed": "The error cause is too large to be displayed",
 	"nodeErrorView.time": "Time",
+	"nodeErrorView.inputPanel.title": "A previous node errored",
+	"nodeErrorView.inputPanel.text": "View error",
```
Comment (mutdmour) [inline packages/editor-ui/src/plugins/i18n/locales/en.json:474]: could we rename these to something more descriptive? maybe something like `nodeErrorView.inputPanel.previousNodeError`

File: packages/editor-ui/src/components/RunData.vue [hunk 2]
```diff
@@ -199,8 +199,14 @@
 				</n8n-text>
 			</div>
 
-			<div v-else-if="hasNodeRun && hasRunError" :class="$style.errorDisplay">
-				<NodeErrorView :error="workflowRunData[node.name][runIndex].error" />
+			<div v-else-if="hasNodeRun && hasRunError" :class="$style.center">
+				<n8n-text v-if="isPaneTypeInput" :class="$style.center" size="xlarge" tag="h3" bold>
```
Comment (mutdmour) [inline packages/editor-ui/src/components/RunData.vue:203]: can use `n8n-heading` component... also size should match other views.. it's too big now 
like here https://github.com/n8n-io/n8n/blob/31b0798b0be7156fe8aa4d62ddc5fcbeda18df3b/packages/editor-ui/src/components/RunData.vue#L223

Comment (cstuncsik) [reply packages/editor-ui/src/components/RunData.vue]: kept `n8n-text` but set the correct size and removed bold so now it looks like the message on the right pane

File: packages/editor-ui/src/components/NodeDetailsView.vue [hunk 3]
```diff
@@ -338,8 +338,8 @@ export default mixins(
 		},
 	},
 	watch: {
-		activeNode(node, oldNode) {
-			if (node && !oldNode && !this.isActiveStickyNode) {
```
Comment (cstuncsik) [inline packages/editor-ui/src/components/NodeDetailsView.vue:342]: @mutdmour do you think it is safe to remove this condition from the watcher? by this, we get an immediate proper update of the components and I'm just guessing this was for external hook invoking and/or telemetry call (so they run only when a user opens the node details view, but now everything is running also when changing node by clicking on the link in the error message)

Comment (mutdmour) [reply packages/editor-ui/src/components/NodeDetailsView.vue:342]: i think it's fine..

Comment (mutdmour) [reply packages/editor-ui/src/components/NodeDetailsView.vue:342]: but test it well :)

**Review by mutdmour — APPROVED** [review]
nice clean solution

**Review by mutdmour — CHANGES_REQUESTED** [review]
Does not seem like you have fixed all issues.. Please fix output panel showing error
Bildschirmfoto 2022-09-19 um 10 18 18

**Review by mutdmour — APPROVED** [review]

**Comment by janober** [comment]
Got released with `n8n@0.195.0`

**Pull request merged**